cocos/renderer/CCQuadCommand.cpp

```diff
@@ -72,9 +72,20 @@ void QuadCommand::init(float globalOrder, GLuint textureID, GLProgramState* glPr
 
 void QuadCommand::reIndex(int indicesCount)
 {
+    // first time init: create a decent buffer size for indices to prevent too much resizing
+    if (__indexCapacity == -1)
+    {
+        indicesCount = std::max(indicesCount, 2048);
+    }
+
     if (indicesCount > __indexCapacity)
     {
+        // if resizing is needed, get needed size plus 25%, but not bigger that max size
+        indicesCount *= 1.25;
```

Comment:
Usually this factor is in range [1.5;2]. Is there any particular reason why 1.25?
https://github.com/facebook/folly/blob/master/folly/docs/FBVector.md